src/src/config.h

```diff
@@ -0,0 +1,39 @@
+#pragma once 
+
+#include <Arduino.h>
+#include "targets.h"
+#include "elrs_eeprom.h"
+
+#define CONFIG_VERSION          1
+
+typedef struct {
```

Comment:
Because a struct is being written and we don't need to track addresses, the variable types can be changed easily in the struct.  Is it worth while dropping them down to uint8_t and reduce the commit time?

Reply:
Replying here for completeness: I don't think its worth a millisecond or two in reducing these types, and it will likely cause pain later when we wanna bump up the size again, and would have to wipe people's settings as part of doing so.